### ISSUE TITLE
Doesn't compile on Mac OS X

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#105]
```
Compile fails it seems, am I doing something wrong here?

$ make
cc -L/usr/local/lib/ -L/opt/local/lib skipfish.c -o skipfish -O3 -Wno-format 
-Wall -funsigned-char -g -ggdb -I/usr/local/include/ -I/opt/local/include/  
-DVERSION=\"1.84b\" \
      http_client.c database.c crawler.c analysis.c report.c -lcrypto -lssl -lidn -lz
http_client.c:39:18: error: idna.h: No such file or directory
http_client.c: In function 'parse_url':
http_client.c:286: warning: implicit declaration of function 'idna_to_ascii_8z'
http_client.c:286: error: 'IDNA_SUCCESS' undeclared (first use in this function)
http_client.c:286: error: (Each undeclared identifier is reported only once
http_client.c:286: error: for each function it appears in.)
report.c: In function 'copy_static_code':
report.c:752: warning: passing argument 3 of 'scandir' from incompatible 
pointer type
make: *** [skipfish] Error 1


$ uname -a
Darwin host.local 8.11.1 Darwin Kernel Version 8.11.1: Wed Oct 10 18:23:28 PDT 
2007; root:xnu-792.25.20~1/RELEASE_I386 i386 i386
```

Original issue reported on code.google.com by `philippe...@gmail.com` on 5 Jan 2011 at 8:47

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Yes, you don't have libidn installed:

http://code.google.com/p/skipfish/wiki/KnownIssues
```

Original comment by `lcam...@gmail.com` on 18 Jan 2011 at 4:47
- Changed state: **WontFix**
